doc/classes/Viewport.xml

```diff
@@ -275,6 +275,9 @@
 		<member name="world_3d" type="World3D" setter="set_world_3d" getter="get_world_3d">
 			The custom [World3D] which can be used as 3D environment source.
 		</member>
+		<member name="xr" type="bool" setter="set_use_xr" getter="is_using_xr" default="false">
+			If [code]true[/code], the sub-viewport will be used in AR/VR process.
```

Comment:
```suggestion
			If [code]true[/code], the viewport will be used in AR/VR process.
```

Reply:
Oops... changed it and expanded on the explanation as well.